TimerTriggeredCalc/AppSettings.cs

```diff
@@ -41,5 +41,17 @@ public class AppSettings
         /// The number of seconds to offset from the top of the minute for the first calculation
         /// </summary>
         public int OffsetSeconds { get; set; }
+
+        /// <summary>
+        /// The username to use when connecting to AF
+        /// An empty string will use the application user
+        /// </summary>
+        public string Username { get; set; }
+
+        /// <summary>
+        /// The password to use when connecting to AF
+        /// An empty string will use the application user
```

Comment:
Should this say password instead of user?

Reply:
Technically, it does not use the password of the application user, but I can see how the wording is confusing. I opted to update the summary and removed that note. Let me know if that is better.